sunpy/net/attr.py

```diff
@@ -142,6 +160,37 @@ def _repr_html_(self):
         return _print_attrs(self, html=True)
 
 
+    def show_in_notebook(self, **kwargs):
+        """
+        Display the attrs tables as interactive grids in a Jupyter Notebook.
+
+        This function utilizes the ``itables`` library to render tables as interactive grids.
+
+        .. note::
+            This function requires the optional dependency ``itables``.
```

Comment:
Can we not sphinx interlink this package in the docs/conf.py?

Reply:
As expected its not working (commit I just made), my guess is because the documentation of itables is in different from the other ones like the home/landing page is a quick starter guide i.e. they don't have "stable" documentation as it is a github.io documentation, I could be wrong here.